code/track.ss

```diff
@@ -28,17 +28,17 @@
   (format #t "checking config for ~a~%" problem)
   (let ((exercisms (lookup 'exercises track-config)))
     (cond ((find (lambda (exercism)
-                   (eq? problem (lookup 'slug exercism)))
-                 exercisms)
-           =>
-           (lambda (config)
-             (unless (assoc 'uuid config)
-               (error 'check-config-for
-                      "please set uuid"
-                      problem))))
-          (else (error 'check-config-for
-                       "please add problem to config/config.ss"
-                       problem)))))
+		   (eq? problem (lookup 'slug exercism)))
+		 exercisms)
+	   =>
+	   (lambda (config)
+	     (unless (assoc 'uuid config)
+	       (error 'check-config-for
+		      "please set uuid"
+		      problem))))
+	  (else (error 'check-config-for
+		       "please add problem to config/config.ss"
+		       problem)))))
```

Comment:
All I did was visit the file... This is why I was against making so many whitespace fixes in the json pull

Reply:
That is terrifying.  Although, all I am seeing are the formatting diff sections, but that is just from this view.

You setup doesn't automatically add trailing whitespace does it?

Reply:
So, why does the formatting look like this now?

You are saying that the formatting in your editor looks like it does on the left, but it looks like the right hand side of the diff here on GitHub?